firebaseClient.ts

```diff
@@ -0,0 +1,25 @@
+import firebase from "firebase";
+
+const firebaseClient = ({ FIREBASE_CONFIG }: any) => {
```

Comment:
shouldn't use `any`, add a tsconfig option (`noImplicitAny: true`) to discourage it